src/style.scss

```diff
@@ -219,18 +228,22 @@ $audio_player_play_btn_triangle_color: $audio_player_base_bg_color !default;
         }
     }
 
-    .audio_controls {
-        // display: -webkit-box;
-        // display: -ms-flexbox;
-        // display: flex;
-        // -webkit-box-orient: horizontal;
-        // -webkit-box-direction: normal;
-        // -ms-flex-direction: row;
-        // flex-direction: row;
-        // margin: 0;
+    >div {
         border-right: 1px solid $border-color;
     }
 
+    // .audio_controls {
```

Comment:
Please don't comment. if it is not required, remove it.

Reply:
Sure will update it!